src/content/guides/loading-and-sending-assets.mdx

```diff
@@ -78,125 +78,3 @@ Clicking on the link will open a page to view and claim the asset transfer.
   }
   ```
   </CodePanel>
-
-## Loading Giftcards
-
-You can load Giftcards by calling our [External Assets](/api/external-assets) endpoint. You will need to use the giftcard number for the `externalId` field. The `pin`, the `issuer` and the `type` need to be on hand too.
```

Comment:
What are we doing with this load external asset endpoint?

Reply:
Tom said to archive it for now as no one is using it and we may want to change it before someone else uses it as it is quite giftcard specific. See thread here https://centrapay.slack.com/archives/C03HXJDPPB4/p1713390769161409